### DISCUSSION TITLE
Pendrive or Windows problem

### DISCUSSION THREAD

**Discussion by angrycsm** [LingmoOS/Community#31] [Bugs Report/缺陷提交]
Good evening everyone, I have a problem with my computer/system.

Well, the problem is that I'm trying to make a bootable pendrive within LingmoOS. I managed to create it, but when I enter the BIOS and select my pendrive as the primary one at startup, it seems that the system automatically starts up, and I can't solve this!

What tips can you give me? As for the ISO I'm using, it's Microsoft's Windows 10, in the NTFS format.

**Comment by elysia-best** [comment]
Have you disabled Secure Boot in your motherboard's BIOS settings? Currently Lingmo OS doesn't support SB. We hope you find this information useful.
